### PR TITLE
Confirm transaction screen - wrap lengthier values

### LINKED ISSUE #12598: Confirm transaction screen - wrapping lengthier value
On confirm transaction screen wrap bigger values as shown below:

Screenshot 2021-11-05 at 5 05 20 PM

### PR DESCRIPTION
**Pull request by jpuri**
Fixes: #12598

Wrap lengthier values on confirm transaction screen:

Screenshot 2021-11-18 at 12 46 46 AM

Screenshot 2021-11-18 at 12 45 59 AM

### CODE REVIEW

File: ui/components/app/transaction-detail-item/index.scss [hunk 1]
```diff
@@ -22,9 +29,10 @@
     }
   }
 
-  &__total {
+  & h6.transaction-detail-item__total {
     font-weight: bold;
     color: $ui-black;
+    margin-left: 4px;
   }
```
Comment (jpuri) [inline ui/components/app/transaction-detail-item/index.scss:36]: Change above is to make this rule more specific.

File: ui/components/app/transaction-detail-item/index.scss [hunk 2]
```diff
@@ -22,9 +29,10 @@
     }
   }
 
-  &__total {
+  & h6.transaction-detail-item__total {
     font-weight: bold;
     color: $ui-black;
```
Comment (NiranjanaBinoy) [inline ui/components/app/transaction-detail-item/index.scss:34]: Hi, I understand that the `font-weight` and `color` might be pre-existing but aren't they already covered in <Typography/> ?

**Review by NiranjanaBinoy — DISMISSED** [review]

File: ui/components/app/transaction-detail-item/index.scss (hunk 1 above)
Comment (mcmire) [reply ui/components/app/transaction-detail-item/index.scss:36]: Hmm. We should be able to write CSS in a way that doesn't require these kinds of specificity overrides. What were you needing to override?

File: ui/components/app/transaction-detail-item/index.scss (hunk 2 above)
Comment (jpuri) [reply ui/components/app/transaction-detail-item/index.scss:34]: yep, it was not noticed as values overriding them were similar.

File: ui/components/app/transaction-detail-item/index.scss (hunk 1 above)
Comment (jpuri) [reply ui/components/app/transaction-detail-item/index.scss:36]: Yep I agree this should ideally not be needed, this classes are styled in couple of components - that makes it hard to fix. Let me spend some more time on it.

Comment (jpuri) [reply ui/components/app/transaction-detail-item/index.scss:36]: It is fixed now and code looks cleaner :)

**Comment by jpuri** [comment]
@mcmire , @NiranjanaBinoy : can you plz re-review the PR ?

**Review by darkwing — APPROVED** [review]

**Review by mcmire — APPROVED** [review]
Looks good!

**Pull request merged**